docs/troubleshooting/large-files.mdx

```diff
@@ -11,31 +11,79 @@ When pushing your changes, you might see a message indicating that some files ex
 
 If you encounter an error like this, it's likely due to inadvertently committing files that are too large or not meant to be tracked (e.g., binaries or `node_modules/`).
 
+Depending on when these large files entered your Git history, there are two ways to resolve this.
 
-## Instructions
-Follow these steps to remove the large files:
+## Solution 1
 
 <Steps>
-  <Step title="Remove the files">
+  <Step title="Stage the large files for deletion">
     Run the following command:
     ```bash
-    git rm paths_to_large_files
+    git rm LARGE_FILE
     ```
-    This command removes the large files from your Git repository.
+    This command removes the large files from your Git repository and locally.
 
     For example to delete the `./server` and `./release/target` binaries shown in the screenshot above, run the following command:
 
     ```bash
-    git rm --cached server release/target
+    git rm server release/target
     ```
 
   </Step>
   <Step title="Commit the changes">
-    After removing all the large files from the GIt repository. commit the changes:
+    ```bash
+    git commit --amend --no-edit --allow-empty
+    ```
+
+    This will amend the previous commit with the changes.
+
+  </Step>
+</Steps>
 
+If Solution 1 doesn't work, it means the large files are present in an older commit and will require some extra work. Follow the instructions in Solution 2 below.
```

Comment:
Unlike below where we mention `git push --force`, there's no such CTA here - maybe it should be an additional Step above?